frontend/apps/reader/modules/readertypography.lua

```diff
@@ -429,6 +431,7 @@ These settings will apply to all books with any hyphenation dictionary.
         end,
         separator = true,
     })
+    table.insert(hyphenation_submenu, self.ui.userhyph:menuEntry())
```

Comment:
`:getMenuEntry()`

Reply:
I don't understand this one.

Reply:
Just that `:getMenuEntry()` reads better than just `:menuEntry()`.
It would be a verb, like your `ReaderUserHyph:addButton`